README.md

```diff
@@ -0,0 +1,54 @@
+# Pronovix Drupal module template
+
+## Start development of a new module
+
+### Building a new module with Composer
+
+```sh
+$ composer create-project -s dev pronovix/drupal-module-template path/to/my_module
+```
+or
+
+```sh
+$ docker run --rm -it -v $PWD:/app --user $(id -u):$(id -g) composer create-project -s dev pronovix/drupal-module-template my_module
+```
+
+See the [composer documentation](https://getcomposer.org/doc/03-cli.md#create-project) for details.
+
+## Contributing to this template
+
+Contributing to this template project is simple.
+
+* Spin up the containers inside the `.dev` folder with `docker-compose up -d`.
```

Comment:
What is your shell? Docker-compose version?

Reply:
Also, run `env | grep COMPOSER_AUTH`

Reply:
Managed to reproduce this during the testing framework demo in a VM so I am investigating this.

Reply:
This should be fixed now.

Reply:
Something is still fishy: when running the same command from above in an empty directory, I'm getting:
```  [Symfony\Component\Filesystem\Exception\IOException]                                                            
  Cannot rename because the target "/app/drupal-module-template/drupal_module_template.info.yml" already exists.  ```

Reply:
OK, it's a PEBKAC on my side: looks like I forgot about pasting the last parameter, the module name. IDK if there should be any sanity check for this kind of (admittedly user) error, tho.